tools/envoy_headersplit/BUILD

```diff
@@ -0,0 +1,46 @@
+load("@rules_python//python:defs.bzl", "py_test")
+load("@headersplit_pip3//:requirements.bzl", "requirement")
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+py_test(
+    name = "headersplit_test",
+    srcs = [
+        "headersplit.py",
+        "headersplit_test.py",
+    ],
+    data = glob(["code_corpus/**"]),
+    main = "headersplit_test.py",
+    python_version = "PY3",
+    srcs_version = "PY3",
+    tags = ["no-sandbox"],
```

Comment:
Is this needed if you are now hermetic?

Reply:
it's still needed since clang module installed with `pip3` still need to be run locally

Reply:
I'm somewhat surprised, but we can take a TODO on this. If we installed under `pip3_import`, I would expect that things work hermetically.

Reply:
Can you add some TODOs to dig into this? I'm still unclear on the interaction with `--spawn_strategy=local`.

Reply:
Yeah, but strangely it will fail to load llvm .so file when running in sandbox. 
I have added a TODO here